tests/kokkos-based/helpers.hpp

```diff
@@ -84,5 +84,235 @@ auto create_stdvector_and_copy_rowwise(T sourceView)
   return result;
 }
 
+// create rank-1 mdspan (vector)
+template <typename ValueType,
+          typename mdspan_t = typename _blas2_signed_fixture<ValueType>::mdspan_r1_t>
+mdspan_t make_mdspan(ValueType *data, std::size_t ext) {
+  return mdspan_t(data, ext);
+}
+
+template <typename ValueType,
+          typename mdspan_t = typename _blas2_signed_fixture<const ValueType>::mdspan_r1_t>
+mdspan_t make_mdspan(const ValueType *data, std::size_t ext) {
+  return mdspan_t(data, ext);
+}
+
+template <typename ValueType>
+auto make_mdspan(std::vector<ValueType> &v) {
+  return make_mdspan(v.data(), v.size());
+}
+
+template <typename ValueType>
+auto make_mdspan(const std::vector<ValueType> &v) {
+  return make_mdspan(v.data(), v.size());
+}
+
+// create rank-2 mdspan (matrix)
+template <typename ValueType,
+          typename mdspan_t = typename _blas2_signed_fixture<ValueType>::mdspan_r2_t>
+mdspan_t make_mdspan(ValueType *data, std::size_t ext0, std::size_t ext1) {
+  return mdspan_t(data, ext0, ext1);
+}
+
+// no-tolerance (exact) comparison
+// TODO: add tolerance based comparison if needed (see is_same_matrix below)
+template <typename ElementType1,
+          typename LayoutPolicy1,
+          typename AccessorPolicy1,
+          typename ElementType2,
+          typename LayoutPolicy2,
+          typename AccessorPolicy2>
+bool is_same_vector(
+    const mdspan<ElementType1, extents<dynamic_extent>, LayoutPolicy1, AccessorPolicy1> &v1,
```

Comment:
The Standard Library convention for view types like `mdspan`, `span`, and `string_view` is to pass them by value, not by const reference.

Reply:
Fixed on https://github.com/kokkos/stdBLAS/pull/187/commits/c0c4e70117366fc32f7d8f41bccff80077610f9c.